### ISSUE TITLE
add typings for `shell.color`

### ISSUE THREAD

**Issue by mweststrate** [mweststrate/nscript#10]

**Comment by morenoh149** [comment]
could you clarify?

**Comment by mweststrate** [comment]
Yes, shell.color namespace can be used to output colored information, like
console.log(shell.color.cyan("Hello world")). But it is currently typed as
'any' so typescript won't auto-complete on the actually available colors.

Op wo 24 feb. 2016 om 17:01 schreef Harry Moreno <notifications@github.com>:
